src/components/container/Container.styles.js

```diff
@@ -0,0 +1,10 @@
+import {StyleSheet} from 'react-native';
+
+export const style = StyleSheet.create({
+    containerDefault: {
```

Comment:
Are you going to use containerDefault in the future?

Reply:
Yeah I think so.